PersonApi.Tests/V1/Gateways/DynamoDbGatewayTests.cs

```diff
@@ -149,5 +157,23 @@ public void GetPersonByIdExceptionThrow()
             mockDynamoDb.Verify(x => x.LoadAsync<PersonDbEntity>(id, default), Times.Once);
             _logger.VerifyExact(LogLevel.Debug, $"Calling IDynamoDBContext.LoadAsync for id {id}", Times.Once());
         }
+
+        [Fact]
+        public async Task UpdatePersonSuccessfulUpdates()
+        {
+            // Arrange
+            var id = Guid.NewGuid();
+            var constructRequest = ConstructRequest(id);
+
+            var dbEntity = constructRequest.ToDatabase();
+
+            // Act
+            await _dynamoDb.SaveAsync(dbEntity).ConfigureAwait(false);
+
+            // Assert
+            var request = ConstructRequest(constructRequest.Id);
+            var response = await _classUnderTest.UpdatePersonByIdAsync(request).ConfigureAwait(false);
+            response.Should().NotBeNull();
+        }
```

Comment:
This doesn't actually test an update being done.
At the moment you 
* Create and save a person to the db
* Call the update gateway method to update the record you just saved, but don't actually change any of the person properties
* Do not validate that any properties were changed in the database after the gateway call.

What you need to do is... (this flow is basically the same as the E2E test):
* Create and save a person to the db
* Change some properties and then use the changed object to call the gateway method.
* After the gateway call, retrieve the person from the db directly and then validate that the new db object has the updated values on it.

Reply:
I've updated the test to do a similar test to the E2E one